### ISSUE TITLE
Unable to login to Google Cloud SDK - ERROR: gcloud failed to load: module 'googlecloudsdk.third_party.apis.secretmanager.v1.secretmanager_v1_messages' has no attribute 'SecretmanagerProjectsSecretsGetIamPolicyRequest'Issue Name

### ISSUE THREAD

**Issue by morfinismo** [GoogleCloudPlatform/cloud-code-vscode#332]
**Environment**:
Extension version: 1.7.0
VSCode version: 1.51.1
OS: win32 - Windows 10

**Description**:

I am unable to login to Google Cloud SDK to view the Kubernetes containers.

**Repro step**:
![vscode_cloud_code_bug](https://user-images.githubusercontent.com/16520045/99160702-8bb59e80-26af-11eb-91b9-131eeef67b6e.gif)

When clicking on "Cloud Code - Kubernetes", in the left panel of VS Code, and then clicking on "Click here to login to Google Cloud SDK", I get the following output:

```
ERROR: gcloud failed to load: module 'googlecloudsdk.third_party.apis.secretmanager.v1.secretmanager_v1_messages' has no attribute 'SecretmanagerProjectsSecretsGetIamPolicyRequest'
    gcloud_main = _import_gcloud_main()
    import googlecloudsdk.gcloud_main
    from googlecloudsdk.command_lib.util.apis import yaml_command_translator
    from googlecloudsdk.command_lib.iam import iam_util
    secrets_message.SecretmanagerProjectsSecretsGetIamPolicyRequest,

This usually indicates corruption in your gcloud installation or problems with your Python interpreter.

Please verify that the following is the path to a working Python 2.7 or 3.5+ executable:
    C:\Users\femor\AppData\Local\cloud-code\installer\google-cloud-sdk\platform\bundledpython\python.exe

If it is not, please set the CLOUDSDK_PYTHON environment variable to point to a working Python 2.7 or 3.5+ executable.

If you are still experiencing problems, please reinstall the Cloud SDK using the instructions here:
    https://cloud.google.com/sdk/

Press any key to exit.
```

**Comment by kschaab** [comment]
@morfinismo Would you mind trying deleting c:\Users\femor\AppData\Local\cloud-code\installer and trying again?  It would be interesting to see if this installation was corrupted in some way or if it is machine specific.  Please let us know so that we can detect this error and take the appropriate action in the future.  Thanks!

**Comment by kschaab** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-vscode-internal](https://github.com/GoogleCloudPlatform/cloud-code-vscode-internal/issues/3601)

**Comment by morfinismo** [comment]
@kschaab Thank you very much for your support. I wish there would be more information that could help determine the cause of the problem; Nonetheless, your suggestion did the trick. 

1. Deleted the folder in the path C:\Users\<user>\AppData\Local\cloud-code\installer
(I also deleted the folder in path C:\Users\<user>\AppData\Local\cloudcode), I don't know if that matters.

2. I uninstalled the extension from VS Code and reloaded it.

3. I installed the extension again and waited for the dependencies to finish intalling.

4. Reloaded VS Code and everything is working fine now.

Huge Thanks!
